torch/_prims/nvfuser_executor.py

```diff
@@ -212,10 +214,18 @@ def nvfuser_execute(gm: GraphModule, *args, executor_parameters=None):
             arg for arg in flat_args if isinstance(arg, (torch.Tensor, Number))
         )
 
-        return tree_unflatten(
+        if os.getenv('PYTORCH_NVFUSER_DUMP_NVTX'): torch.cuda.nvtx.range_push("fusion: {0}, graph: {1}".format(fusion.id(),
+                str([{
+                    'op': n.op,
+                    'name': n.name,
+                    'args': n.args
+                } for n in gm.graph.nodes])))
+        result = tree_unflatten(
             fusion.execute(concrete_fusion_inputs),  # type: ignore[has-type]
             unflatten_spec,  # type: ignore[has-type]
         )
+        if os.getenv('PYTORCH_NVFUSER_DUMP_NVTX'): torch.cuda.nvtx.range_pop()
```

Comment:
Let's save `os.getenv('PYTORCH_NVFUSER_DUMP_NVTX')` outside this function call, no need to query `os` for every execution of this function.